lib/mayaUsd/utils/diagnosticDelegate.cpp

```diff
@@ -104,12 +111,13 @@ void UsdMayaDiagnosticDelegate::IssueError(const TfError& err)
 {
     // Errors are never batched. They should be rare, and in those cases, we
     // want to see them separately.
-    // In addition, always display the full call site for errors by going
-    // through _FormatDiagnostic.
+
+    const auto diagnosticMessage = _FormatDiagnostic(err);
+
     if (ArchIsMainThread()) {
-        MGlobal::displayError(_FormatDiagnostic(err));
+        MGlobal::displayInfo(diagnosticMessage);
```

Comment:
Was it an intention change (or a copy/paste mistake) that you always call displayInfo() even for the error/warning?

Reply:
okay.